crates/symbolicator/src/services/symcaches.rs

```diff
@@ -390,6 +390,7 @@ fn write_symcache(
                 handle.uuid,
                 object_handle
             );
+            #[allow(deprecated)]
```

Comment:
we deprecated an api we ourselves keep using? 😂

Reply:
:-D it was a bit overeager, I agree. We should migrate to symcache transformers. But that requires unsafe SelfCell code for now. So creating an `OwnedBcSymbolMap` type or something inside symbolic that is `'static` will solve that.